test/Sentry.Tests/Internals/ProcessInfoTests.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Threading.Tasks;
+using Sentry.Internal;
+using Sentry.Testing;
+using Xunit;
+
+namespace Sentry.Tests.Internals
+{
+    public class ProcessInfoTests
+    {
+        [Fact]
+        public async void SetupStartupTime_StartupTimeSet()
```

Comment:
If you return `void` xunit can't await for your test.
```suggestion
        public async Task SetupStartupTime_StartupTimeSet()
```

Does it fail if you change `Assert.True` to `Assert.False`?

Reply:
It should fail since it returned True.

Reply:
TIL